testhelper/testhelper.go

```diff
@@ -51,14 +54,22 @@ var mockLidarPath = artifact.MustPath("viam-cartographer/mock_lidar")
 
 // SetupStubDeps returns stubbed dependencies based on the camera
 // the stubs fail tests if called.
-func SetupStubDeps(cameraName string, t *testing.T) resource.Dependencies {
+func SetupStubDeps(cameraName, movementSensorName string, t *testing.T) resource.Dependencies {
 	deps := make(resource.Dependencies)
 	switch cameraName {
 	case "stub_lidar":
 		deps[camera.Named(cameraName)] = getStubLidar(t)
 	default:
 		t.Errorf("SetupStubDeps called with unhandled camera: %s", cameraName)
 	}
+	switch movementSensorName {
+	case "stub_imu":
+		deps[movementsensor.Named(movementSensorName)] = getStubIMU(t)
+	case "":
+		return deps
```

Comment:
[nit] Can remove this line as the return at the end of the function is the same.

Reply:
Just realized that integration tests without IMU fail if we do not allow for an empty IMU - this is because the default is set to error if it does not recognize the IMU we passed in, so we need to allow both stub_imu and an empty string (i.e. no IMU)

Reply:
talked in person